googledrive/src/main/java/ch/cyberduck/core/googledrive/DriveAttributesFinderFeature.java

```diff
@@ -104,7 +104,7 @@ public PathAttributes toAttributes(final File f) {
         if(null != f.getExplicitlyTrashed()) {
             if(f.getExplicitlyTrashed()) {
                 // Mark as hidden
-                attributes.setDuplicate(true);
+                attributes.setHidden(true);
```

Comment:
Has been changed from `hidden` to `duplicate` and now `hidden` again - might lead to a regression?

Reply:
Has test coverage .